Chisel/CHLFunctions.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CHLFunctions.h
+//  Chisel
+//
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <malloc/malloc.h>
+
+/**
+ * Enumerates all live memory allocations by calling `block` for each malloc range (represented as a `vm_range_t`).
+ */
+void CHLEnumerateAllocationsWithBlock(void (^block)(vm_range_t range));
```

Comment:
@modocache What do I add here to indicate the block can't be null? Should I add an assert internally too?